### DISCUSSION TITLE
Changing mission name/ disable missions

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1540] [Question and answer] [answered]
Hello i have 2 questions:
- Is it possible to change a mission name
- Is it possible to disable some missions?
Cheers.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Is it possible to change a mission name

Are you talking about side mission?

> Is it possible to disable some missions?

If you are talking about side mission, yes: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L229
You can remove any of them from the array
Please, change this array carefully

Cheers
